chef-provisioning-docker.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.homepage = 'https://github.com/opscode/chef-provisioning-docker'
 
   s.add_dependency 'chef'
-  s.add_dependency 'chef-provisioning', '~> 0.9'
+  s.add_dependency 'chef-provisioning', '~> 1.0'
```

Comment:
It's pessimistic, so it will retrieve the latest version in the 1.* scope.
```
Gem::Requirement.new("~> 1.0").satisfied_by?(Gem::Version.new("1.2.0"))
```